### PR TITLE
ROMFS: updated deltaquad autoconfig params

### PR DESCRIPTION
**Pull request by RomanBapst**
Based on latest flight testing and input from @sanderux

### CODE REVIEW

**Comment by RomanBapst** [comment]
Logs:
Position Control + RTL auto landing: https://logs.px4.io/plot_app?log=9620f092-8372-43c7-b2d7-42214201af0a
Mission: https://logs.px4.io/plot_app?log=7f415549-df2a-4bf1-8d9d-9d282efc21e2

**Comment by RomanBapst** [comment]
@sanderux I want to actively control pitch during the front transition to keep the target height. It's not difficult and should be more robust than just having a static offset.

**Comment by sanderux** [comment]
even better.
perhaps you should also add the MC/FW PID params

File: ROMFS/px4fmu_common/init.d/13013_deltaquad [hunk 1]
```diff
@@ -84,28 +100,45 @@ then
...
+	param set MPC_TILTMAX_AIR 25
+	param set MPC_TILTMAX_LND 25
 
 	param set CBRK_AIRSPD_CHK 162128
 	param set CBRK_IO_SAFETY 22027
 
 	param set EKF2_GPS_POS_X -0.12
 	param set EKF2_IMU_POS_X -0.12
 	param set EKF2_TAU_VEL 0.5
+	param set EKF2_GPS_DELAY 110
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/13013_deltaquad]: This is the default value.

**Comment by RomanBapst** [comment]
@sanderux I think I did. Did I miss any? I did not lower the mc rollrate P because it was flying great with no sign of oscillations.

**Pull request merged**